pkg/clients/has/components.go

```diff
@@ -533,36 +530,6 @@ func (h *HasController) StoreAllComponents(namespace string) error {
 	return nil
 }
 
-// specific for tests/remote-secret/image-repository-cr-image-pull-remote-secret.go
-func (h *HasController) CreateComponentWithoutGenerateAnnotation(componentSpec appservice.ComponentSpec, namespace string, secret string, applicationName string, skipInitialChecks bool) (*appservice.Component, error) {
```

Comment:
Why was this one removed?

Reply:
Because it is not used anymore. Leftover after removal of RemoteSecret tests for Image Controller